src/stores/configStore.js

```diff
@@ -115,6 +123,9 @@ export class ConfigStore {
   @observable minLocalCounts = initialConfigValues.minLocalCounts;
   @observable minGlobalCounts = initialConfigValues.minGlobalCounts;
 
+  @observable urlParams = initialConfigValues.urlParams;
+  @observable locationArray = initialConfigValues.locationArray;
```

Comment:
The `@observable` tag is a MobX feature that allows UI components to "listen" to these store variables and re-render when a change is detected. Since these variables will probably never have UI components listening to them, we can remove the decorator and just have them as normal class attributes